pkg/build/builder/source.go

```diff
@@ -434,7 +434,10 @@ func extractSourceFromImage(ctx context.Context, dockerClient DockerClient, stor
 	if err != nil {
 		return err
 	}
-	capabilities := defaultContainerConfig.Capabilities("", nil, dropCapabilities())
+	capabilities, err := defaultContainerConfig.Capabilities("", nil, dropCapabilities())
+	if err != nil {
+		return err
+	}
```

Comment:
@otaviof this come from the buildah bump?

Reply:
Yes, that's correct. And the reason for `buildah` bump explained below.